api/registrations/views.py

```diff
@@ -96,10 +109,27 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
+    def is_blacklisted(self, query):
+        for query_param in query.nodes:
```

Comment:
So then I get the public_field and I check to see if it is wrapped in HideIfRetraction.  If it is, I know this is a blacklisted field.   If I am filtering on a blacklisted field, I want to exclude retractions (this was @felliott's suggestion, which I thought was pretty good).  You don't want them to be able to guess information about retractions through filtering.